editor/editor_inspector.cpp

```diff
@@ -3205,6 +3206,7 @@ void EditorInspector::update_tree() {
 
 		// If we did not find a section to add the property to, add it to the category vbox instead (the category vbox handles margins correctly).
 		if (current_vbox == main_vbox) {
+			category_vbox->set_visible(true);
```

Comment:
Not doing `is_visible()` check because `set_visible()` already does it

Reply:
May be worth putting this statement in a on the side comment (if it's deemed a bit odd).